src/lib/zendesk/zendesk.client.ts

```diff
@@ -0,0 +1,98 @@
+import logger from "../logger";
+
+const zendesk = require('node-zendesk')
+
+const zendeskClient = zendesk.createClient({
+  username: process.env.ZENDESK_USER,
+  token: process.env.ZENDESK_API_KEY,
+  remoteUri: process.env.ZENDESK_URL,
+  proxy: process.env.http_proxy
+})
+
+export async function updateTicketWithWorldpayGoLiveResponse(ticket: ZendeskTicket, requester: ZendeskUser, worldpayGoLiveUrl: string): Promise<boolean> {
+  logger.info(`Updating zendesk ticket ${ticket.id} with go live response...`)
+  try {
+    await zendeskClient.tickets.update(ticket.id,
+      {
+        ticket: {
+          // custom_fields: [{ id: 360012004639, value: "something"}], <-- This can be done in the future which would enable us to automatically close the ticket
+          comment: {
+            body: `
+Dear ${requester.name}
+
+You’re nearly ready to start taking payment on GOV.UK Pay. You just need to link your GOV.UK Pay account with your payment service provider’s (PSP) account.
+
+You’ll find guidance about how to do this for your PSP in our documentation at https://docs.payments.service.gov.uk/switching_to_live/#switching-to-live
+
+Click this link to go to the live version of your service: ${worldpayGoLiveUrl}
```

Comment:
Will this be a hyperlink or do we need to add markup to make it a link?

Reply:
No need markup. You can see that zendesk automatically makes it a link at my [test ticket](https://govuk.zendesk.com/agent/tickets/5284419).